cds-snc/cppd-medical-report
# Translate Section 6 to French

# Description

[Trello card:](https://trello.com/c/a46aCfHK/309-people-can-access-prototype-in-en-and-fr) Translates section 6 of the medical report into French

# Test Instructions

1. Go to the application (from the linked "View Deployment")
1. Change the URL to `.../fr/frsection6` from `.../en/section6`)
1. See French translation (original EN and FR forms are linked in the [Trello card](https://trello.com/c/a46aCfHK/309-people-can-access-prototype-in-en-and-fr))

# Help Requested

If there are any things you spot that are issues we should come back to, but are out of scope of just the direct translation, please add a new Trello card or [use this one to quickly add a note to the checklist](https://trello.com/c/WqJVeqsh/356-misc-content-tasks). I've already added a few from section 6 to that card, around the conditional questioning.
